src/color.js

```diff
@@ -473,7 +473,8 @@ export default class Color {
 
 			let fallbacks = Array.isArray(fallback)? fallback.slice() : Color.defaults.fallbackSpaces;
 
-			for (let i = 0, fallbackSpace; fallbackSpace = fallbacks[i]; i++) {
+			for (let i = 0; fallbacks[i]; i++) {
+				let fallbackSpace = fallbacks[i];
```

Comment:
Why?

Reply:
This is from [`no-cond-assign`](https://eslint.org/docs/rules/no-cond-assign), and personally I agree with its judgement here. I am not a big fan of doing assignments in conditions.

Reply:
I have no strong opinion about this particular case, but I wouldn't want to blanket forbid assigning in conditionals.